test/TestProjects/FirstTest-TypeSpec/tests/Generated/Samples/Samples_FirstTestTypeSpecClient.cs

```diff
@@ -760,6 +760,50 @@ public async Task Example_StringFormat_Convenience_Async()
             var result = await client.StringFormatAsync(Guid.NewGuid(), body);
         }
 
+        [Test]
+        [Ignore("Only validating compilation of examples")]
+        public void Example_HeadAsBoolean()
```

Comment:
Do we have any cadl-ranch test cases?  We should verify whether this API do return true/false.

Reply:
Yes. I am working on add cadl-ranch test cases.